assets/airports/kbos.json

```diff
@@ -1,713 +1,8666 @@
-        "SHAAN": ["N42.57990100260009", "W70.82768318454781"],
-        "SHAUN": ["N42.67622768922109", "W70.64076911800313"],
-        "SOBAH": ["N42.23733435243515", "W70.14538641685365"],
-        "SOFEE": ["N41.84317584404800", "W71.25891488333254"],
-        "SPYSD": ["N42.44969786960377", "W71.02708503898530"],
-        "SSAAM": ["N42.63223229311426", "W71.38600838661355"],
-        "SSOXS": ["N41.83679841823984", "W70.74610969645450"],
-        "STAYC": ["N42.50778089801794", "W70.79162971697076"],
-        "STEEM": ["N42.24206878950581", "W70.82566902434802"],
-        "STOLI": ["N42.74090505128394", "W73.11456067019694"],
-        "SUFFI": ["N43.02493494624861", "W69.91718206621232"],
-        "TAALE": ["N42.62623051498790", "W70.77189094701240"],
-        "TEKKK": ["N42.41271802017828", "W71.09863808543068"],
-        "TJAYY": ["N42.30499929589529", "W70.96041634171699"],
-        "TKMAN": ["N42.58190314953434", "W71.18086617558893"],
-        "TONNI": ["N42.37332190330239", "W70.61353832585554"],
-        "TTERI": ["N42.52960379229147", "W70.51528693755337"],
-        "VOCUS": ["N42.45408196815536", "W70.96192695116805"],
-        "WAATR": ["N42.45779707939299", "W70.82556831633800"],
-        "WACHU": ["N42.65238219377580", "W71.48107674805044"],
-        "WHATE": ["N42.43602344084837", "W72.76167980316850"],
-        "WIDUP": ["N42.48112286969136", "W71.16153022699130"],
-        "WINNI": ["N42.11717719593455", "W71.12446967931068"],
-        "WINTA": ["N42.50347479630538", "W71.36667244869417"],
-        "WYANE": ["N42.53042013761245", "W70.92476569547746"],
-        "WYMTH": ["N42.21220233023077", "W70.82707892577585"],
-        "WYMTH": ["N42.21220233992196", "W70.82717964449787"],
-        "YAARD": ["N42.22234583011669", "W70.83996955105606"],
-        "ZOMIB": ["N42.53546645852862", "W71.23494636628250"],
-        "ZUMOL": ["N42.40517069730649", "W70.98559333351842"]
-
+    {
+      "name": ["22R", "4L"],
+      "end": [
+        ["N42d22m41.89", "W71d00m16.39", "15ft"], 
+        ["N42d21m28.08", "W71d00m51.75", "14ft"]
+      ],
+      "ils": [false, false]
+    },
+    {
+      "name": ["22L", "4R"],
+      "end": [
+        ["N42d22m36.89", "W70d59m57.57", "15ft"], 
+        ["N42d21m03.86", "W71d00m42.59", "19ft"]
+      ],
+      "ils": [true, true]
+    },
+    {
+      "name": ["27", "9"],
+      "end": [
+        ["N42d21m36.83", "W70d59m15.86", "15ft"], 
+        ["N42d21m20.76", "W71d00m46.54", "17ft"]
+      ],
+      "ils": [true, false]
+    },
+    {
+      "name": ["14", "32"],
+      "end": [
+        ["N42d21m23.08", "W71d01m23.92", "16ft"], 
+        ["N42d20m55.01", "W71d00m29.81", "19ft"]
+      ],
+      "ils": [false, false]
+    }
+  ],
+  "sids": {
+    "BLZZR3": {
+      "icao": "BLZZR3",
+      "name": "Blazer Three",
+      "rwy": {
+        "4R":  ["NHANT", ["HURBE", "A40+"], "ORRRR", "CHWDH", "STEEM", "DEDHM"],
+        "9":   [["CLAWW", "A40+"], "FENWY", "GLLET", "WYMTH", "DEDHM"],
```

Comment:
Single spacing only, except in the definitions of non-underscore-prefixed fixes.